openapi.yml

```diff
@@ -1267,7 +1266,6 @@ components:
           type: string
         properties:
           $ref: "#/components/schemas/source-form-properties"
-      required: []
```

Comment:
what was the validation error for this?

Reply:
```
connexion.exceptions.InvalidSpecification: {'description': 'Request body for updating a source', 'content': {'application/json': {'schema': {'$ref': '#/components/schemas/update-source-body', 'x-scope': ['']}}}, 'required': True} is not valid under any of the given schemas

Failed validating 'oneOf' in schema['properties']['paths']['patternProperties']['^/']['properties']['put']['properties']['requestBody']:
    {'oneOf': [{'$ref': '#/definitions/requestBody'},
               {'$ref': '#/definitions/reference'}]}

On instance['paths']['/v4/sources/{source_id}']['put']['requestBody']:
    {'content': {'application/json': {'schema': {'$ref': '#/components/schemas/update-source-body',
                                                 'x-scope': ['']}}},
     'description': 'Request body for updating a source',
     'required': True}
```

Reply:
if you take out this line instead https://github.com/NarrativeScience/stitch-connect-openapi/blob/master/openapi.yml#L238 will it pass validation? i think that would be more accurate

Reply:
ah nope, still fails with the same exception.

Reply:
oh, rip, i'm not sure what the correct way to denote that none of the keys are required for that request body. but it seems fine to take that out